templates/profiles/my_profile.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load markdownify %}
+{% load courses_extra %}
+{% load bootstrap_tags %}
+
+
+{% block content %}
+    <form method="post">
+  {% csrf_token %}
+          <section class="bg-light">
+    <div class="py-5 container">
+      <div class="row">
+        <div class="col-12 mx-auto">
+          <p>{{ form|markdownify }}</p>
+        </div>
+      </div>
+    </div>
+  </section>
+{#  {{ form }}#}
```

Comment:
Form variable is there twice (even this is commented), please remove the commented bits in final code.